CHANGELOG.md

```diff
@@ -37,6 +37,22 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Checksum values calculated by `@cumulus/checksum` are now converted to string to standardize
   checksum formatting across the Cumulus library.
 
+### Added
+
+### Changed
+
+- **CUMULUS-2718**
+  - Updated config for SyncGranules to support optional `workflowStartTime`
+  - Updated SyncGranules to provide `createdAt` on output based on `workflowStartTime` if provided,
+  falling back to `Date.now()` if not provided.
+  - Updated cookbook_browse_example_workflow.asl.json,
```

Comment:
You only need to say  something like 'Updated task config of SyncGranule in example workflows'